doozerlib/distgit.py

```diff
@@ -584,8 +584,9 @@ def push_image(self, tag_list, push_to_defaults, additional_registries=[], versi
 
                     with io.open(push_config, 'w', encoding="utf-8") as pc:
                         pc.write('\n'.join(all_push_urls))
-
                     mirror_cmd = 'oc image mirror --filter-by-os=amd64 {} {} --filename={}'.format(dr, insecure, push_config)
```

Comment:
aiieeee! unbelievable, i found another problem with this plan.

this only mirrors the amd64 arch; and we need that for 3.11 because reg-aws can't handle manifest lists.
but for OCP 4 operators we need to mirror the manifest list and all arches. so in that case i think we should omit the `--filter-by-os` flag.

if doozer needs changes to add a namespace translation to the push, we could also add an option that specifies limiting the mirror to amd64 in 3.11, and pushing the manifest list otherwise.

Reply:
It would be good if 3.11 job could use a separate version of doozer, but I think I can add a flag like `--filter-by-os` as well.